makerdao/governance-portal-v2
# #809 Refresh votes when changing account

### Link to Shortcut ticket:

https://app.shortcut.com/dux-makerdao/story/809/refresh-votes-when-changing-account

### What does this PR do?

Updates vote selections on polling page when an account changes 

### Steps for testing:

1. Go to polling page
2. Notice state of the votes
3. Swtich to an account with different vote selections
4. See updated votes ✨ 

### Add a GIF:
![](https://media.giphy.com/media/oJ8gRS9lwBGpHiQaZC/giphy.gif)
